src/cf.data.pre

```diff
@@ -2570,7 +2570,16 @@ DOC_START
 	Allows you to apply a Netfilter mark value to packets being transmitted
 	on the client-side, based on an ACL.
 
-	clientside_mark mark-value [!]aclname ...
+	clientside_mark mark[/mask] [!]aclname ...
+
+	Mark and mask are unsigned integers (hex, octal, or decimal).
+	The mask may be used to preserve the mark previously set by other agents
+	(e.g., iptables).
+
+	The directive clears the original connmark's bits given by the mask and ORs
+	it with a new mark. The result is set as a new connmark.
```

Comment:
The first sentence is too complex (because it is trying to translate code into English) and also a little buggy (we do not OR with the _whole_ new mask). I suggest simplifying by focusing on the code result (not the process) and assuming that the two sentences will be interpreted together (instead of trying to talk about the mask effect in each sentence). Something like this may work better (shown without formatting):

> A matching rule replaces the original connmark with the configured one. If a mask is specified, then only the masked bits are affected.

I also suggest adding a paragraph to document handling of multiple rules:

> The directive rules are evaluated in the configuration order. The first matching rule wins.